core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -1349,6 +1357,10 @@ class GroupMetadataManagerTest {
     val topicPartition = new TopicPartition("foo", 0)
     val offset = 37
 
+    val sensor = groupMetadataManager.offsetCommitsSensor
+    val offsetCommitCount = new WindowedCount()
+    sensor.add(metrics.metricName("commit-count", "unit-test", "offset commit count"), offsetCommitCount)
```

Comment:
Out of curiosity, why do we need a new metric? Can we not reuse `offset-commit-count`?

Reply:
It seems that we can't get a metric from `offsetCommitsSensor` or `metrics`, only `Sensor` has a method `synchronized List<KafkaMetric> metrics()` which can get all metrics but it's package-private, do you think it's worthwhile to add a method in `Sensor` just for testing?

Reply:
Hmm, we have some utilities in `TestUtils`. See for example `totalMetricValue`. Could we not do something like that?

Reply:
Thank you for your reminders, I add a method another `totalMetricValue` in `TestUtils` to improve this.